dcos/packagemanager.py

```diff
@@ -658,6 +531,9 @@ def _format_error_message(error):
         error_message = _format_json_schema_mismatch_message(error)
     elif error.get("type") == "MarathonBadResponse":
         error_message = _format_marathon_bad_response_message(error)
+    elif error.get('type') == 'NotImplemented':
+        error_message = 'DC/OS has not been ' \
+                        'configured to support this operation: package add'
```

Comment:
lets remove `package add` from the message. I assume this error type will be try for other operations as well